beetsplug/smartplaylist.py

```diff
@@ -201,6 +209,7 @@ def update_playlists(self, lib):
                 if item_path not in m3us[m3u_name]:
                     m3us[m3u_name].append(item_path)
 
+        started_with = bytestring_path(self.config['prefix'].get())
```

Comment:
`get` should be `as_str` here to enforce a type on the configuration. And can we call this variable `prefix`?